worker/worker.go

```diff
@@ -168,6 +173,36 @@ func (w *Worker) EmitEvent(event ipc.WorkerEventName, message string) error {
 	return w.conn.Emit(dbus.ObjectPath(path.Join("/com/redhat/Yggdrasil1/Worker1", w.directive)), "com.redhat.Yggdrasil1.Worker1.Event", args...)
 }
 
+func (w *Worker) cancel(addr string, id string, cancelID string) *dbus.Error {
+	// If worker doesn't implements the cancelation function it does nothing
+	if w.cancelRX == nil {
+		log.Debug("worker doesn't support cancellation messages")
+		return nil
+	}
+
```

Comment:
I would add here log message introducing following log messages to give them some context. Something like `log.Tracef("cancelling message")`.

Reply:
And I added such message (https://github.com/RedHatInsights/yggdrasil/pull/128/files#diff-a497cf86b016d9c95cc8d6a4d99bf1bea36490b188fad3414822222234b654aeR250) ;-)

Reply:
Looks like this is settled by the presence of a `log.Tracef` line on 185.